spec/src/consensus.rs

```diff
@@ -23,17 +23,21 @@ pub(crate) const MEDIAN_TIME_BLOCK_COUNT: usize = 11;
 
 //TODO：find best ORPHAN_RATE_TARGET
 pub(crate) const ORPHAN_RATE_TARGET_RECIP: u64 = 20;
+// dampening factor
+pub(crate) const TAU: u64 = 2;
 
-const MAX_BLOCK_INTERVAL: u64 = 30 * 1000; // 30s
-const MIN_BLOCK_INTERVAL: u64 = 8 * 1000; // 8s
+const MAX_BLOCK_INTERVAL: u64 = 30; // 30s
+const MIN_BLOCK_INTERVAL: u64 = 8; // 8s
 const TWO_IN_TWO_OUT_CYCLES: Cycle = 2_580_000;
-pub(crate) const EPOCH_DURATION_TARGET: u64 = 4 * 60 * 60 * 1000; // 4 hours
+pub(crate) const EPOCH_DURATION_TARGET: u64 = 4 * 60 * 60; // 4 hours, unit: second
+pub(crate) const MILLISECOND_TO_SECOND: u64 = 1000;
```

Comment:
You get 0.001 second when converting a millisecond to second. 

```suggestion
pub(crate) const MILLISECONDS_IN_A_SECOND: u64 = 1000;
```